packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowCheck.tsx

```diff
@@ -46,6 +47,14 @@ const DetailsRowCheckBase = (props: IDetailsRowCheckProps) => {
     compact
   });
 
+  const defaultCheckboxRender = (checkboxProps: IDetailsCheckboxProps) => {
+    return <Check checked={checkboxProps.checked} />;
```

Comment:
I'd consider hoisting this function if you can so its not within the render path and constantly mutating. My understanding could be incorrect w.r.t performance tho.

Reply:
If its not re-evaluated then disregard the above